.github/workflows/test-server.yml

```diff
@@ -54,7 +55,7 @@ jobs:
           POSTGRES_USER: postgres
           POSTGRES_DB: fires_test
         options: >-
-          --health-cmd pg_isready
+          --health-cmd "pg_isready -U ${POSTGRES_USER}"
```

Comment:
This fixes the noise in the `Stop containers` section of the GitHub Actions workflow logs, where something like `FATAL:  role "root" does not exist`